### ISSUE TITLE
Problem with make on Raspberry Pi

### ISSUE THREAD

**Issue by TimRustige** [sui77/rc-switch#75]
Current master version download appears to have RCSwitch.o missing? Make on Raspberry PI fails with:

pi@raspberrypi ~/lloytron/433Utils/RPi_utils $ make
make: *** No rule to make target '../rc-switch/RCSwitch.o', needed by 'send'.  Stop.
pi@raspberrypi ~/lloytron/433Utils/RPi_utils $

**Comment by JerryWorkman** [comment]
Get the source for RCSwitch here: https://github.com/r10r/rcswitch-pi.git

**Comment by fingolfin** [comment]
@TimRustige No, RCSwitch.o is a binary object file, which the C/C++ compiler generates from the source code; as such it should not be in the repository.

Since you are executing a "make" command inside of `433Utils/RPi_utils`, I'd guess that this is a problem (?) with the build system of that project -- or perhaps you missed a step in their build instructions which explains how to compile RCSwitch?

**Comment by fingolfin** [comment]
@JerryWorkman Uhm, I don't think it's a good idea to point people to a 4 year old unmaintained fork of RCSwitch.

**Comment by huongminh** [comment]
I came across the same issue trying to run "make". then I tried rotv/433Utils and it worked. Thank you!

**Comment by utsxumiao** [comment]
@rotv is spot on!

**Comment by ovidiu-pislaru** [comment]
go to the root folder of your project. do a `git submodule init` and `git su`

**Comment by NickSee** [comment]
thanks  ovidiu-pislaru this also worked for me in terms of getting the make to complete correctly.

**Comment by jppuertolas** [comment]
thanks, it also worked for me for building the make
